app/views/users/_edit_form.html.erb

```diff
@@ -1,24 +1,25 @@
 <div style="padding:10px;">
-<%= form_for @user, :url => {:controller => 'users', :action => 'new'}, :html => {:class => "row"} do |f| %>
-<% if f.error_messages != "" %><div class="alert alert-danger"><%= f.error_messages %></div><% end %>
-
+<%= form_for @user, :url => {:controller => 'users', :action => 'update'}, :html => {:class => "row"} do |f| %>
   <div class="col-sm-10">
+    <% if f.error_messages != "" %>
+      <div class="alert alert-danger"><%= f.error_messages %></div>
+    <% end %>
 
-      <div class='col-sm-12' style="display:flex; justify-content: center;">
-        <h3> Link your Public Lab account to providers </h3>
-      </div>
+    <div class='col-sm-12' style="display:flex; justify-content: center;">
+      <h3> Link your Public Lab account to providers </h3>
+    </div>
```

Comment:
Please make these indentation back to original version so that it is not shown in git

Reply:
there is an if statement above it I just added "|| params[:action] == "update" " to make sure the save button appears once someone enters a wrong password confirmation or invalid email